components/http_foundation/introduction.rst

```diff
@@ -483,6 +483,9 @@ abstracts the hard work behind a simple API::
 
     $response->headers->set('Content-Disposition', $d);
 
+You can add the file's content as usual, for example by using
```

Comment:
@raphaelm setting aside the grammar/wording issues ... the problem I see with the original text is that it says: *"well, if you want, include the file contents"*. But it should say something like: *"don't forget that it's mandatory to include the content files and you must do that manually; otherwise, the file is not really sent"*.